satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,25 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		id
+
+	field	id				serial
+	field	user_id user.id	restrict
+	field	offer_id offer.id	restrict
+	field	referred_by user.id	restrict
+	field	credits_earned	int
+	field	credits_used	int
+	field	credits_type	int
```

Comment:
nit pick: this can be `credit_type`

Reply:
Why did you delete credit_type from the model ?

Reply:
By looking at the `referred_by`, if it's nil, we know it's not an invitee credit, if it has value, we know it's a referral invitee credit. That's all we care about right now. So we don't have use of the `credit_type`

Reply:
Gotcha